syntax = 'proto3';

message Features {
    float row_Number     = 1 ;
    string user_review   = 2 ;
    float polarity       = 3 ;

}

message Empty {
}

service get_next_row {
    rpc get_next_row(Empty) returns(Features);
}